File: challenges/stockpicker/js/stockPicker.js
const picker = (prices) => {
    let dates = []
    let largestProfit = 0
    for (let i = 0; i < prices.length; i++) {
        for (let j = i; j < prices.length; j++) {
            if (prices[j] - prices[i] > largestProfit) {
                largestProfit = prices[j] - prices[i]
                dates = [i, j]
            }
        }
    }
    return dates
}

module.exports = { picker }

File: challenges/stockpicker/python/stock_picker.py
def picker(prices):
    largest_profit = 0
    for i in range(len(prices)):
        for j in range(i, len(prices)):
            if prices[j] - prices[i] > largest_profit:
                largest_profit = prices[j] - prices[i]
                dates = [i, j]
    return dates
